utils_nlp/models/transformers/common.py

```diff
@@ -90,6 +93,7 @@ def fine_tune(
         max_grad_norm=1.0,
```

Comment:
the device arg is not needed here anymore. The device should be created here

Reply:
revomed.